README.md

```diff
@@ -293,6 +292,50 @@ document.body.appendChild(document.createElement('iframe')).contentWindow.fetch(
 
 The conclusion is that not being able to enforce such services at a very specific time (earlier than all other scripts) to all same origin realms of the app (as opposed to its top most realm only), significantly narrows down the reach such services have, which attackers can abuse.
 
+## Security
+
+This section explores this proposal's impact on security from the perspective of the well known [CIA](https://agoric.com/blog/technology/a-taxonomy-of-security-issues) (Confidentiality,Integrity,Availability) security standard.
+
+### Integrity
+
+Reading through the different sections of this proposal makes it clear RIC is a security feature that aspires to support integrity specifically.
+
+As described in this document, same origin realms are a power that can allow untrusted code that's invited to run within the same context of a given web app to undermine its will to apply its own set of rules and restrictions on its hosting envirnoment.
+
+While such rules and restrictions are applicable via JavaScript runtime virtualization, that statement does not hold against the power to form same origin realms, which is untamable.
+
+RIC aspires to provide web apps the power to tame it, by allowing them to execute their code within such realms before any other hosted code, so that they can apply their rules and restrictions within them too.
```

Comment:
```suggestion
RIC aspires to provide web apps the power to tame it, by allowing them to execute their code within such realms before any other hosted code, so that they can apply their rules and restrictions within these realms too.
```